keyboards/vitamins_included/keymaps/vitavim/keymap.c

```diff
@@ -0,0 +1,104 @@
+#include QMK_KEYBOARD_H
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _ALPHAS 0
+#define _MODS 1
+#define _VIM 2
+#define _NUMPAD 3
+#define _CODE 4
```

Comment:
We recommend using enums for the layer definitions, as they're easier to maintain, long term. 

```suggestion
enum layers {
    _ALPHAS,
    _MODS,
    _VIM,
    _NUMPAD,
    _CODE,
};
```